tools/run_tests/xds_k8s_test_driver/framework/infrastructure/k8s.py

```diff
@@ -234,8 +236,8 @@ def wait_for_deployment_available_replicas(
             self,
             name,
             count=1,
-            timeout_sec=WAIT_LONG_TIMEOUT_SEC,
-            wait_sec=WAIT_LONG_SLEEP_SEC):
+            timeout_sec=WAIT_MEDIUM_TIMEOUT_SEC,
+            wait_sec=WAIT_MEDIUM_SLEEP_SEC):
```

Comment:
The PR title says "increase timeouts for k8s cleanup operations" but this change reduces timeout and is not for a cleanup operation. The bug also was only for delete timeouts. Just making sure this change is intended and trying to understand the context.

Reply:
That's because `WAIT_LONG_TIMEOUT_SEC` essentially got renamed to `WAIT_MEDIUM_TIMEOUT_SEC`. See description for the details

Reply:
Yes I noticed it right away and modified my comment :-)